bundles/org.openhab.ui/web/src/pages/settings/pages/sitemap/__tests__/sitemap-edit_jest.spec.js

```diff
@@ -186,39 +350,4 @@ describe('SitemapEdit', () => {
     wrapper.vm.validateWidgets()
     expect(lastDialogConfig).toBeFalsy()
   })
-
-  it('validates valuecolor', async () => {
```

Comment:
Why did you remove that test?

Reply:
It shouldn't have been removed. I don't know what happened there. I put it back in.